plugins/item_previews/web_client/templates/itemPreviews.jade

```diff
@@ -0,0 +1,24 @@
+.g-widget-item-previews-header
+  i.icon-eye
+  |  Item Previews
+
+.g-widget-item-previews-container
+
+  each item in items
+
+    .g-widget-item-preview
+
+      - var name = item.get('name')
+
+      - if (isImageItem(item))
+        a.g-item-preview-link(g-item-name="#{name}")
```

Comment:
Jade tip: rather than `g-item-name="#{name}"`, you can use the shorthand `g-item-name=name`.